src/user/newsletter/_newsletter-subscription.scss

```diff
@@ -0,0 +1,12 @@
+.newsletter-subscription-page {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
I think that's not necessary, should be part by component

Reply:
resolved in pair programming !